src/components/alert/__spec__.js

```diff
@@ -12,7 +12,7 @@ describe('Alert', () => {
   beforeEach(() => {
     wrapper = shallow(
       <Alert
-        open
+        open={ false }
```

Comment:
why'd you change this? It looks like it has affected the snapshot

Reply:
When open is left as the default (set to true) it ends up calling modalHTML in Dialog (which Alert inherits from (which is actually called from the Modal render() action)) which uses a 'ref', which shallow doesn't have access to. 
The two paths appeared to be either:  Use mount rather than shallow for the alert, but then we'd be snapshotting a mount, or set open to false so it doesn't attempt to render the Alert out. 
It looks like this is not causing the issue though of we're not actually testing the Alert in a meaningful way via. the snapshot any more.
Would you suggest we actually snapshot the Alert but with mount rather than shallow in this case? Or have any other suggestions of how to move forward with these ones?

Reply:
Actually we've changed  these back to having the Alert open.